util/cmd/bump_version/main.go

```diff
@@ -45,19 +45,27 @@ func main() {
 			}
 
 			if bumpMajor || bumpMinor || bumpPatch {
+				if !oneof(bumpMajor, bumpMinor, bumpPatch) {
+					log.Fatalf("Expected only one of --major, --minor, and --patch.")
+				}
 				versions := strings.Split(CURRENT_RELEASE, ".")
-
 				major, _ := strconv.Atoi(versions[0])
```

Comment:
I think it might be worth checking errors just in case. If you (understandably) don't want error handling logi all over the place, something like this should help:

```go
atoi := func(s string) int {
  i, err := strconv.Atoi(s)
  if err != nil {log.Fatal(err)}
  return i
}

major := atoi(versions[0])
// ...
```

Reply:
Done but moved to the bottom to be consistent with btoi.